docs/BuildFromSource.md

```diff
@@ -222,7 +222,7 @@ code .
 
 ### Building on command-line
 
-When developing in VS Code, you'll need to use the `build.cmd` or `build.sh` scripts in order to build the project. You can learn more about the command line options available, check out [the section below](using-dotnet-on-command-line-in-this-repo).
+When developing in VS Code, you'll need to use the `eng\build.cmd` or `eng\build.sh` scripts in order to build the project. You can learn more about the command line options available, check out [the section below](using-dotnet-on-command-line-in-this-repo).
```

Comment:
I think this is talking about individual folders, so the previous statement was correct

Reply:
Suggest reordering the verbiage to introduce the "project areas" before talking about the build scripts in those areas. Basically, everything after this point should be about building for an area though we could call out using eng/build.sh and eng/build.cmd as a special case when you need to build the whole repo.

nit: never use backslashes when talking about build.sh scripts